src/Solver/Interoperability.cpp

```diff
@@ -408,7 +408,9 @@ double c_interoperability_M2invDiagonal(int no) {
 
   extern void f_interoperability_setFrictionOutputStrength(void*  i_domain, int i_face, real* strength);
 
-  extern void f_interoperability_setFrictionOutputInitialStress(void*  i_domain,
+extern void f_interoperability_setFrictionOutputThermalPressurization(void*  i_domain, int i_face, real* fluidPressure, real* fluidTemperature);
+
+extern void f_interoperability_setFrictionOutputInitialStress(void*  i_domain,
                                                                 int i_face,
                                                                 real* i_initialStressInFaultCS,
                                                                 real* i_bulkXX,
```

Comment:
Please, check the indentation. Make sure that it matches with the rest of the file. I would suggest to write all arguments in a new line as follows:
```
extern void f_interoperability_setFrictionOutputThermalPressurization(void*  i_domain,
                                                                      int i_face,
                                                                      real* fluidPressure,
                                                                      real* fluidTemperature);

```